.github/workflows/codegen.yml

```diff
@@ -48,7 +50,7 @@ jobs:
 
       - name: Commit changes
         run: |
-          git config --local user.email "codegen@backengine.dev"
+          git config --local user.email ${{ github.event.inputs.commit_email }}
```

Comment:
pedantically other inputs are set as env vars, can we do either or for consistency, and whilst there I think GH_TOKEN is unused?